### ISSUE TITLE
Missing white space between inline header and content

### ISSUE THREAD

**Issue by manuel489** [metanorma/metanorma-iec#61]
````
[%inline-header]
==== Balance,

capable of weighing to the nearest 0,01 g.
````

renders as

![inline-header](https://user-images.githubusercontent.com/33627611/83214048-a90ea480-a131-11ea-8149-5c792b1d16cb.PNG)

White space is missing.

**Comment by opoudjis** [comment]
Right: I inject a tab (space in HTML) after the clause number, but I need to do so again if a title exists as well.
